includes/widgets/video.php

```diff
@@ -802,7 +802,17 @@ protected function render() {
 
 			$embed_options = $this->get_embed_options();
 
-			$video_html = Embed::get_embed_html( $video_url, $embed_params, $embed_options );
+			$is_static_render_mode = Plugin::$instance->frontend->is_render_mode( Frontend::RENDER_MODE_STATIC );
+			$post_id = get_queried_object_id();
+
+			if ( $is_static_render_mode ) {
+				// Make sure that the oembed data saved in cache.
+				Embed::get_oembed_data( $video_url, $post_id );
```

Comment:
Actually you don't need this call, since the `Embed::get_embed_thumbnail_html` do it

Reply:
Right, my mistake.